### ISSUE TITLE
metric comparison

### ISSUE THREAD

**Issue by geoffwoollard** [flatironinstitute/Cryo-EM-Heterogeneity-Challenge-1#113]
- [ ] information imbalance
- [ ] MDS: https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html
- [ ] pearson correlation

**Comment by geoffwoollard** [comment]
![Image](https://github.com/user-attachments/assets/25e42de5-213d-49a0-a3a1-44f13ee50a6b)

![Image](https://github.com/user-attachments/assets/9ec92022-463d-4b88-9a87-67eaba28eeae)

![Image](https://github.com/user-attachments/assets/d381ef2e-bf17-422b-b599-78e677e73393)

**Comment by geoffwoollard** [comment]
![Image](https://github.com/user-attachments/assets/6994b02c-a2b8-4077-863b-e3e44be99150)

**Comment by geoffwoollard** [comment]
![Image](https://github.com/user-attachments/assets/6c965752-c61c-4e69-82eb-28f4611e0a25)

![Image](https://github.com/user-attachments/assets/f21cc132-1c75-4b38-a16d-05f96bd79ba9)

![Image](https://github.com/user-attachments/assets/9daa90de-0d61-48a5-9d09-1491736a3bf9)
